### ISSUE TITLE
Crash when loading after settings change

### ISSUE THREAD

**Issue by SnaipyLeo** [MartinTheDragon/Nuclear-Tech-Mod-Remake#66]
### Issue description

Crash when changing settings like minmap levels, resource pack or any other settings that causes the mojang loading screen to appear, after forge update.
When tried to remove mods to see which one caused it, found out that only without HBM, this bug does not reappear.

### Steps to reproduce

1. Open Settings (in menu or in game)
2. Change the texture pack or minmap level setting.
3. after the mojang loading screen game crashes.

### Forge version

40.2.4

### Nuclear Tech version

0.0.20

### (Crash) Log

https://pastebin.com/buXCS5Wu

**Comment by MartinTheDragon** [comment]
Hi there!
Thanks for reporting, but this is very likely an incompatibility with some other mod's incorrect implementation of armor materials. Issues with armor materials have happened several times, even with the Twilight Forest mod, though not quite something like this issue in particular.
See if removing mods that add armor items (or anything you put into the armor slot) solves the issue. Add them back in and tell me which one seems to be causing it (unless you don't want to, in which case, do tell).
Again, thanks for reporting!

**Comment by SnaipyLeo** [comment]
Hello again!
removing the create 0.5.1c mod and it's addons (Create Crafts & Additions, Create: Crystal Clear, Create: Steam & Rails and Create Deco) fixed the issue with the crash, but still when removing hbm crash doesn't appear as well, so looks like it's either an incompatibility of hbm with create or it's addons, i will check if removing the addons one by one fixes the issue.

**Comment by SnaipyLeo** [comment]
Just found out that it was Create: Steam & Rails that was causing the crash, even after updating it to the newest version, this still reappears, so it's the addon's issue with hbm for some reason, but yeah ;)

**Comment by MartinTheDragon** [comment]
The conductor cap returns `null` as the repair ingredient instead of `Ingredient.EMPTY`, which is a bad implementation 100% on Create: Steam'n'Rails' side. NTM uses repair ingredients as a fallback to calculate radiation resistance for armor, so other mods don't have to register radiation resistance themselves. Putting the cap in a vanilla anvil would crash the game if the cap was damageable.

```java
@Override
public Ingredient getRepairIngredient () {
    return null;
}
```
[Source](https://github.com/Layers-of-Railways/Railway/blob/f8c7c93493f4e4c1176f4923dce64b49096d5d1a/common/src/main/java/com/railwayteam/railways/content/conductor/ConductorCapItem.java#L87)

~~Please report the issue to their [issue tracker](https://github.com/Layers-of-Railways/Railway/issues). You can reference this issue if you want.~~

Thanks for testing!

**Comment by MartinTheDragon** [comment]
I reported the issue to them (<https://github.com/Layers-of-Railways/Railway/issues/253>). You don't need to report it anymore.

**Comment by MartinTheDragon** [comment]
Alright, they fixed the issue on 1.19, but you might have to wait for a 1.18 backport. You could do it yourself, or also just ask them to, in case they forget.

**Comment by SnaipyLeo** [comment]
Thank you very much! You’re the best!
